src/services/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+const API_KEY = "zoZERMgHqLkGo92C0JJiJV7RY8PCmmkN";
```

Comment:
I would advise keeping your key on a `.env` file not versioned and provide an `.env.example` to express the usage of the var name.

Reply:
Sure, I was going to ask you about that, is there any dependency that could help me with that? I tried by placing it into a .env file, and call it in my api.js file with process.env.API_KEY but it didn't work. I have used this dependency in react-native https://github.com/zetachang/react-native-dotenv is there something similar for web?

Reply:
I found a workaround, using this prefix `REACT_APP_` I don't know if this is the standard